openapi/types.rs

```diff
@@ -0,0 +1,138 @@
+/// Types used for REST communication with the svc-cargo server
+
+use serde::{Deserialize, Serialize};
+use utoipa::{IntoParams, ToSchema};
+use chrono::NaiveDateTime;
+
+/// API Endpoint to Cancel a Flight
+pub const ENDPOINT_CANCEL: &str = "/cargo/cancel";
+
+/// API Endpoint to Confirm a Flight
+pub const ENDPOINT_CONFIRM: &str = "/cargo/confirm";
+
+/// API Endpoint to Query for Available Flights
+pub const ENDPOINT_QUERY: &str = "/cargo/query";
+
+/// API Endpoint to Get Vertiports for a Region
+pub const ENDPOINT_REGION: &str = "/cargo/region";
+
+/// Request Body Information for Flight Query
+#[allow(dead_code)]
+#[derive(Debug, Clone, IntoParams, ToSchema)]
+#[derive(Deserialize, Serialize)]
+pub struct FlightQuery {
+    vport_depart_id: String,
+    vport_arrive_id: String,
+    timestamp: NaiveDateTime,
+    weight_kg: f32,
+}
+
+impl FlightQuery {
+    /// Creates a new flight query with required fields
+    /// # Arguments
+    /// vport_depart_id: The String ID of the vertiport to leave from
+    /// vport_arrive_id: The String ID of the destination vertiport
+    /// timestamp: The delivery time
+    /// weight_kg: The approximate weight of the cargo
+    #[allow(dead_code)]
```

Comment:
What does it mean by "The delivery time"? Like an ETA? Or the time at which a customer wants a cargo aircraft to depart?

Reply:
Fixed, changed to:
- depart_timestamp_min
- depart_timestamp_max

A "window" of departure

Reply:
https://github.com/Arrow-air/svc-cargo/pull/1/commits/9cdbe864c593acdc4bf89bcc76db1d2aa2e97227